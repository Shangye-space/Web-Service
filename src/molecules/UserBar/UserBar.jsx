import React from 'react';

import styles from './UserBar.module.scss';

import Cart from '../../atoms/Cart';
import Account from '../../atoms/Account';

const UserBar = () => {
    return (
        <div className={styles.userBarWrapper}>
            <Cart />
            <Account />
        </div>
    );
};

export default UserBar;
