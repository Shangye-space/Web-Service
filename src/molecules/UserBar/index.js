import UserBar from './UserBar.jsx';

export default UserBar;
