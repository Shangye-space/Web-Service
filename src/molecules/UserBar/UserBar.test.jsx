import React from 'react';
import ReactDOM from 'react-dom';
import UserBar from './UserBar.jsx';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<UserBar userData={{ name: 'Test' }} />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<UserBar userData={{ name: 'Test' }} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
