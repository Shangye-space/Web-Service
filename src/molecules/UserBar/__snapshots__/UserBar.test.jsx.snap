// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="userBarWrapper"
>
  <div
    className="cartWrapper"
  >
    <a
      className="fas fa-shopping-cart"
    />
  </div>
  <div
    className="accountWrapper"
    onClick={[Function]}
  >
    <div
      className="fas fa-user-circle"
    />
    <span>
      Hi, 
      Test
      !
    </span>
  </div>
</div>
`;
